portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,116 @@
-#content {
+/* General styling */
+
+html {
+  scroll-behavior: smooth;
+}
+
+h1 {
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+}
+
+/* Website welcome styling */
+
+#welcome-content {
   margin-left: auto;
```

Comment:
Here and below, alphabetize declarations.

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order